src/default.nix

```
@@ -200,6 +200,7 @@
   # fetch only sources and do not build
   fetchSources = {
     dreamLock,
+    sourceRoot,
```

This should be optional.

```suggestion
    sourceRoot ? null,
```